routes/Laundry/get_status.js

```diff
@@ -1,4 +1,4 @@
-module.exports = function(req, res){
-	var status = ['Available', 'In Use', 'Almost Done', 'End of Cycle', 'Out of Order', 'Offline', 'Ready To Start'];
+module.exports = function (req, res) {
+	let status = ['Available', 'In Use', 'Almost Done', 'End of Cycle', 'Out of Order', 'Offline', 'Ready To Start'];
```

Comment:
Not a fix you need to make, but I want to find a way to make these dynamic as well :/